app/SideCiSample6.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class SideCiSample6 {
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)